src/pudl/metadata/codes.py

```diff
@@ -1,6 +1,10 @@
 """Metadata for cleaning, re-encoding, and documenting coded data columns.
 
-These dictionaries are used to create Encoder instances. They contain the following keys:
+These dictionaries are used to create Encoder instances. Each key is a table name with
+a sub dictionary that includes additional detail. The table names must end with the
+data_source as a sufix (for EIA 860, 861 or 923 tables include ``_eia``).
```

Comment:
@zaneselvans  can we use pydantic for enforce this?

Reply:
I don't think I understand enough to comment right now. Maybe we can chat tomorrow?

Reply:
Okay I think I understand now and made a comment on the PR in the `etl.py` module changes where you're loading the static tables.